blank/webpack.config.js

```diff
@@ -1,54 +1,73 @@
 const path = require('path')
-const ImageminPlugin = require('imagemin-webpack')
-const { CleanWebpackPlugin } = require('clean-webpack-plugin')
+const webpack = require('webpack')
 const CopyPlugin = require('copy-webpack-plugin')
-const HtmlWebpackPlugin = require('html-webpack-plugin')
 const TerserPlugin = require('terser-webpack-plugin')
+const HtmlWebpackPlugin = require('html-webpack-plugin')
+const { CleanWebpackPlugin } = require('clean-webpack-plugin')
+const ImageMinimizerPlugin = require('image-minimizer-webpack-plugin')
 
-const webpack = require('webpack')
-
-const root = path.resolve(__dirname, 'src')
-const botonicPath = path.resolve(__dirname, 'node_modules', '@botonic', 'react')
-
-const terserPlugin = new TerserPlugin({
-  parallel: true,
-  sourceMap: true,
-  terserOptions: {
-    keep_fnames: true,
-  },
-})
+const ROOT = path.resolve(__dirname, 'src')
+const OUTPUT_PATH = path.resolve(__dirname, 'dist')
+const BOTONIC_PATH = path.resolve(
+  __dirname,
+  'node_modules',
+  '@botonic',
+  'react'
+)
 
-const MODE_DEV = 'development'
-const MODE_PROD = 'production'
+const WEBPACK_MODE = {
+  DEVELOPMENT: 'development',
+  PRODUCTION: 'production',
+}
 
 const BOTONIC_TARGETS = {
+  ALL: 'all',
   DEV: 'dev',
   NODE: 'node',
   WEBVIEWS: 'webviews',
   WEBCHAT: 'webchat',
 }
 
+const WEBPACK_ENTRIES = {
+  DIR: 'webpack-entries',
```

Comment:
I would place this value outside WEBPACK_ENTRIES dictionary since it's the folder's name and not one of the entries.

Reply:
done 👍